tools/run_tests/python_utils/jobset.py

```diff
@@ -130,15 +130,15 @@ def message(tag, msg, explanatory_text=None, do_newline=False):
         try:
             if platform_string() == 'windows' or not sys.stdout.isatty():
                 if explanatory_text:
-                    logging.info(explanatory_text)
+                    logging.info(explanatory_text.decode('utf8'))
```

Comment:
Cleanup like this should be a separate PR.
Making it separate would make things much easier to review and would protect us from back and forth in case we need to rollback you proto generation changes.